### PR TITLE
Arjav Changes for JWT

### PR DESCRIPTION
**Pull request by aarjavshah**
The changes for integration eith JWT.

### CODE REVIEW

File: src/middleware/createProfile.js [hunk 1]
```diff
@@ -24,16 +24,18 @@ const createProfile = store => next => async action => {
...
     formData.append("candidate", candidateInformation.details);
 
-    const data = await fetch(`${constants.host}/candidates/create`, {
-     
-      method: "POST",
-      body: formData
-    }).then(data => data.json());
+    const config = {
+      method: 'post',
+      url: `/candidates/create`,
```
Comment (CGIBIFE) [inline src/middleware/createProfile.js]: create Routes constants for all urls

File: src/AxiosClient.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    return config;
+})
+
+instance.interceptors.response.use(
+  function (response) {
+    return response;
+}, function (error) {
+    if (401 === error.response.status) {
+      store.dispatch(setGeneralError("Session Expired. Please login again."));
```
Comment (CGIBIFE) [inline src/AxiosClient.js]: Please use constant for all strings, this would because full when we do translation

File: src/AxiosClient.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import {logout} from "./actions/userActions";
+import store from "./store"
+import { setGeneralError } from "./actions/errorActions";
+
+const instance = axios.create({
+    baseURL: constants.host
+  });
+
+instance.interceptors.request.use((config)=>{
+    const token = sessionStorage.getItem("authToken");
```
Comment (CGIBIFE) [inline src/AxiosClient.js]: I think you have missed this string.

Comment (aarjavshah) [reply src/AxiosClient.js]: changes done.

**Review by CGIBIFE — APPROVED** [review]

**Pull request merged**